src/engraving/compat/midi/pitchwheelrenderer.cpp

```diff
@@ -100,13 +102,14 @@ void PitchWheelRenderer::renderChannelPitchWheel(EventMap& pitchWheelEvents,
             for (const auto& pitch : pitches) {
                 finalPitch += pitch;
             }
-            if (finalPitch != prevPitch || tick == start) {
+            if (forceUpdate || finalPitch != prevPitch || tick == start) {
```

Comment:
Technically there might be two and more notes with same pitch wheel function following one by one. The will have the same pitch value and as a result not update pitch. Because we're resetting pitch value before every note on event, all notes after the first one won't have pitch wheel events. This is how we handle it.